boot-core/src/daemon/contract.rs

```diff
@@ -89,28 +103,48 @@ where
         Ok(wasm_code_path)
     }
 
-    /// Calculate the checksum of the wasm file to compare against previous uploads
```

Comment:
Ok, ill use those, thanks for the head up!

in vs code It will show the first comment on top of whatever you are hovering (on mouse enter) if you press command (for mac), wont toggle if you already have the mouse over

but yea those are different from doc comments